README.md

```diff
@@ -7,7 +7,23 @@
 ## Installation
 
 - Download the binary for your platform from [the releases page](https://github.com/purescript/psc-package/releases), and copy it somewhere on your PATH, or
-- Build from source, using `stack install`. 
+- Build from source, using `stack install`.
+
+## Usage
+
+```shell
+# install the dependencies listed in psc-package.json
+$ psc-package install
```

Comment:
Is this right?

Reply:
Oh you changed it, I see.

Reply:
I'd rather not have two ways to do the same thing, could you please move that to an issue or separate PR for discussion?

Reply:
My mistake. I mentioned it in https://github.com/purescript/psc-package/issues/63#issuecomment-336160024 and thought you had agreed. I'll submit a separate PR.